src/cargo/ops/vendor.rs

```diff
@@ -297,7 +297,7 @@ fn sync(
                     // Can be removed once the minimal supported Windows version gets bumped.
```

Comment:
Would you mind having another commit explaining why the fallback needed, and remove the part of "Can be removed once the minimal supported Windows version gets bumped"?

Here is a list in may mind:

* In Windows 10 earlier than 1607 the destination of `fs::rename` can't be a directory.
* `from` and `to` are on different file system
* AntiVirus or our system indexer are doing stuf simutaneously
* Any other reasons documented in `std::fs::rename`